.travis.yml

```diff
@@ -2,7 +2,8 @@ language: scala
 jdk:
   - oraclejdk8
```

Comment:
Ahhh, that is a good point!  There is definitely a difference between compile and runtime.  Maybe there is some travis magic we can do to run the server and client tests at JDK 6 and the java-it-tests at JDK8.  Does that sound like a good route?  I can look into that.